2-Browsers/Week1/homework/ex4-whatsTheTime/index.js

```diff
@@ -7,6 +7,60 @@
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
   // TODO complete this function
+  const date = new Date();
+  let h = date.getHours();
+  let m = date.getMinutes();
+  let s = date.getSeconds();
+  let session = 'AM';
+
+  if (h === 0) {
```

Comment:
Are you sure a AM/PM clock is supposed to show 0.00h as 12.00h AM?

Reply:
Yes, you are right. I changed the code.
` // if (h === 0) {
  //   h = 12;
  // }`